Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -109,6 +110,37 @@ async function main(): Promise<void> {
         });
         var artifacts = [];
 
+        // Clean destination folder if requested
+        if (cleanDestinationFolder) {
+            console.log(tl.loc('CleaningDestinationFolder', downloadPath));
+
+            // stat the destination folder (downloadPath) path
+            let destinationFolderStats: tl.FsStats;
+            try {
+                destinationFolderStats = tl.stats(downloadPath);
+            }
+            catch (err) {
```

Comment:
```suggestion
          } catch (err) {
```
Could you please fix the code formatting?

Reply:
Resolved